modmesh/pilot/_gui.py

```diff
@@ -97,6 +98,7 @@ def launch(self, name="pilot", size=(1000, 600)):
         self.burgers = _burgers1d.Burgers1DApp(mgr=self._rmgr)
         self.linear_wave = _linear_wave.LinearWave1DApp(mgr=self._rmgr)
         self.profiling = _profiling.Profiling(mgr=self._rmgr)
```

Comment:
The variable name should be changed to `openprofiledata` in this PR, like:

```python
self.openprofiledata = _profiling.Profiling(mgr=self._rmgr)
```

The `Profiling` class should also be renamed to be `OpenProfileData`, but not in this PR, because it will be too large a change.  The large changed should use a different PR.  I created an issue #575 to track the work.